docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-start.md

```diff
@@ -36,6 +36,13 @@ subcommand: start
   {{ name.usage }}
   </div>
 </pre>
+{% assign note = name.notes %}
+{{ note.description }}
+<pre class="codeblock-pre">
```

Comment:
@alphaX86 will you be removing this from the PR?

Reply:
This can be left in-place. It was just unclear to me what this is / was.

Reply:
Yeah, everyone may get confused due to that YML data import there. I should have done it manually